spec/acceptance/policy/rules/create_rules_spec.rb

```diff
@@ -37,5 +37,15 @@
         expect(page).to have_content "There is a problem"
       end
     end
+
+    context "when there is an existing fallback policy" do
+      let!(:policy) { create(:policy, fallback: true) }
+
+      it "creates a new rule" do
```

Comment:
This probably should say that it does not create a new rule. Fallback policies cannot have rules associated with them.

```suggestion
      it "does not create a new rule" do
```